python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/dhcp-servers.j2

```diff
@@ -16,6 +17,11 @@
 {%         do port_channel_interfaces_dhcp_server.append(port_channel_interface) %}
 {%     endif %}
 {% endfor %}
+{% for vlan_interface in vlan_interfaces | arista.avd.natural_sort('name') %}
```

Comment:
You don't need the natural sort up here since you also natural sort during the printout. You can just use an `| arista.avd.default([])` which requires much less processing.

Reply:
please update on the two others too. Thanks.

Reply:
Fixed!